src/unix/fs.c

```diff
@@ -889,45 +888,6 @@ static ssize_t uv__fs_write(uv_fs_t* req) {
 }
 
 static ssize_t uv__fs_copyfile(uv_fs_t* req) {
-#if defined(__APPLE__) && !TARGET_OS_IPHONE
-  /* On macOS, use the native copyfile(3). */
-  static int can_clone;
-  copyfile_flags_t flags;
-  char buf[64];
-  size_t len;
-  int major;
-
-  flags = COPYFILE_ALL;
```

Comment:
By the way, if this is ever restored for some reason, I imagine this should *probably* be `COPYFILE_DATA` rather than `COPYFILE_ALL`. If the file already exists, I imagine the expected behavior is that its mode will be left intact rather than changed to match that of the source file.

Reply:
I think `COPYFILE_ALL` is still what we would want, but it won't be an option anytime soon anyway.